src/tests/FunctionalTests/Android/Device_Emulator/LibraryMode_AOT_LLVM/Program.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+public static class Program
+{
+    [UnmanagedCallersOnly(EntryPoint = nameof(SayHello))]
+    public static void SayHello()
+    {
+        Console.WriteLine("Called from native!  Hello!");
+    }
+
+    public static int Main()
+    {
+        Console.WriteLine("Hello, Android!"); // logcat
+        return 42;
+    }
```

Comment:
Is the `Main()` necessary for libraries tests. I think we don't have that for iOS ones https://github.com/dotnet/runtime/blob/3f3af5ee9b5012103f8ef7f87892ac72d9b269c1/src/tests/FunctionalTests/iOS/Device/LibraryMode/ClassLibrary.cs#L9-L17

Reply:
Yes, because library builder relies on the fact that the output type is `Exe`, so a Main entry point must exist.